setup.cfg

```
@@ -51,7 +51,7 @@ docs =
     sphinx-astropy
 all =
     scikit-image
-    jwst<=1.1.0
+    jwst>=1.2
```

```suggestion
    jwst>=1.2.1
```

If we were pinning the max version because we were waiting on a fix in 1.2.1, should we pin this to at least 1.2.1 rather than just 1.2 right?

---

Good point. Didn't know why I did that. Monday Morning ™️ .

---

p.s. Though the CI still passed but doesn't hurt to bump one more bugfix version.